rust/gui-client/src-tauri/src/client/gui.rs

```diff
@@ -260,6 +250,11 @@ pub(crate) fn run(cli: &client::Cli) -> Result<(), Error> {
                     .await
                 });
 
+                // Clean up
+                if let Err(error) = client::resolvers::revert().await {
```

Comment:
Actually I think this can go into connlib easily. Then it will be symmetrical, if `tun_linux.rs` both configures and reverts DNS.

I just need to implement Drop for `Tun`, which is already done on Windows. Since Linux didn't have it, I had assumed it was impossible.